main.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'app'
+
+class Options
+  def initialize
+    @app = App.new(self)
+    puts 'WELCOME TO THE SCHOOL LIBRARY APP!!'
+    show_menu
+  end
+
+  def show_menu
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - Create a rental'
+    puts '3 - Create a person'
```

Comment:
Great job @stino-x 🤝

- [ ] As requested by the previous reviewer, kindly ensure the app is able to create a person. At the moment the app is unable to create the person.
![create student](https://github.com/stino-x/PST-Classes/assets/63917725/f2d4a1cc-5fd7-4b12-9256-fd1b96a10b4f)